src/raw/unix.rs

```diff
@@ -357,6 +376,32 @@ impl Socket {
             }
         }
     }
+    #[cfg(not(any(target_os = "linux", target_os = "freebsd", target_os = "openbsd", target_os = "dragonflybsd")))]
+    pub fn accept(&self, flags: c_int) -> io::Result<(Socket, net::SocketAddr)> {

```

Comment:
Please add docs description to your function.

On first line short meaningful description.

After empty line please add notes on usage(which flags are acceptable)